examples/pip_install/.bazelrc

```diff
@@ -1 +1,2 @@
-test --test_output=errors
+# https://docs.bazel.build/versions/main/best-practices.html#using-the-bazelrc-file
+try-import %workspace%/user.bazelrc
```

Comment:
This change seems good but also unrelated to this PR. Should it be its own?

Reply:
Does it matter hugely if this is included? I was a way for me to test the changes since the examples are already a little difficult to run. I found this to be necessary in validating the changes were safe and would opt to leave it in.

Reply:
It doesn't matter hugely. In cases where we have to revert functionality it can matter, because this is a feature we want regardless of this PR.